CHANGELOG.md

```diff
@@ -1,5 +1,62 @@
+| `primitive-square`       |                           | Need this icon? Let us know by [opening an issue](https://github.com/primer/octicons/issues/new?assignees=&labels=icon+request&template=icon-request.md&title=%5BIcon+request%5D) |
+| `radio-tower`            | `broadcast`               |                                                                                                                                                                                   |
+| `repo-clone`             |                           | Need this icon? Let us know by [opening an issue](https://github.com/primer/octicons/issues/new?assignees=&labels=icon+request&template=icon-request.md&title=%5BIcon+request%5D) |
+| `repo-force-push`        | `repo-push`               |                                                                                                                                                                                   |
+| `repo-pull`              |                           | Need this icon? Let us know by [opening an issue](https://github.com/primer/octicons/issues/new?assignees=&labels=icon+request&template=icon-request.md&title=%5BIcon+request%5D) |
+| `repo-template-private`  |                           | Need this icon? Let us know by [opening an issue](https://github.com/primer/octicons/issues/new?assignees=&labels=icon+request&template=icon-request.md&title=%5BIcon+request%5D) |
+| `request-changes`        | `diff`                    |                                                                                                                                                                                   |
+| `ruby`                   |                           | Need this icon? Let us know by [opening an issue](https://github.com/primer/octicons/issues/new?assignees=&labels=icon+request&template=icon-request.md&title=%5BIcon+request%5D) |
+| `saved`                  | `bookmark`                |                                                                                                                                                                                   |
+| `squirrel`               |                           | Need this icon? Let us know by [opening an issue](https://github.com/primer/octicons/issues/new?assignees=&labels=icon+request&template=icon-request.md&title=%5BIcon+request%5D) |
+| `text-size`              | `heading` or `typography` |                                                                                                                                                                                   |
+| `unsaved`                | `bookmark-slash`          |                                                                                                                                                                                   |
```

Comment:
Do we want to mention `triangle-...` has been renamed to `caret-...`?

Reply:
Good catch! I'll add that. Thank you :)